docs/source/_templates/layout.html

```diff
@@ -9,23 +9,23 @@
   }}();
 </script> -->
 
-<title>EvalML - Alteryx Open Source AutoML Library</title>
-<meta name="description" content="EvalML is an AutoML library that builds, optimizes, and evaluates machine learning pipelines using domain-specific objective functions. — EvalML documentation">
+<title>CheckMate - Alteryx Open Source AutoML Library</title>
+<meta name="description" content="CheckMate is an AutoML library which catches and warns of problems with your data and problem setup before modeling. - CheckMate documentation">
 
 <!-- Facebook Meta Tags -->
-<meta property="og:url" content="https://evalml.alteryx.com/en/stable/">
+<meta property="og:url" content="https://checkmate.alteryx.com/en/stable/">
```

Comment:
I don't think there's anything at this site? We can comment this out for now

Reply:
commented out